App.js

```diff
@@ -1,24 +1,80 @@
-import React from 'react'
+import React, { useState } from 'react'
 import styled from 'styled-components/native'
 
 const Container = styled.View`
   flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
   align-items: center;
+  background: papayawhip;
 `
 
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
+const Card = styled.Image`
+  flex: 1;
+  width: 200px;
+  height: 200px
+  resizeMode: contain;
+  margin: 5px
+`
+const Text = styled.Text`
+  align-items: center;
+  justify-content: center;
+  color: white;
+`
+const FetchButton = styled.TouchableOpacity`
+  background-color: tomato;
+  align-items: center;
+  justify-content: center;
+  fontWeight: bold;
+  color: white;
+  width: 100px;
+  height: 50px;
+`
+const TextField = styled.TextInput`
+height: 40;
+width: 200;
+margin: 5px;
+borderColor: tomato;
+borderWidth: 3;
+textAlign:center;
 `
 
 const App = () => {
+
+  const [fetched, setFetched] = useState(false)
+  const [value, onChangeText] = useState('Fireball');
```

Comment:
This pair should rather be named `value` and `setValue`

Reply:
fixed